.github/workflows/nightly.yml

```diff
@@ -524,9 +524,12 @@ jobs:
       - name: dependencies
         run: |
           apt update
+          # Workaround. Remove when https://discourse.ros.org/t/preparing-for-rolling-sync-delayed-by-python-3-10-transition/24521 solved
+          apt install -y python3.9-dev
```

Comment:
```suggestion
```
Should this be backported?

Reply:
I'm not sure if the `.yml` file that is executed is the one in the default branch aka `galactic`... So we idk if we are able to have a different scripts for each branch.